distribution/changelog.txt

```diff
@@ -21,6 +21,8 @@
 - Fix: [#5893] Looking at guest window tabs other than the main tab eventually causes assertion.
 - Fix: [#5905] Urban Park merry-go-round has entrance and exit swapped (original bug).
 - Fix: [#6006] Objects higher than 6 metres are considered trees (original bug).
+- Fix: [#7039] Map window not rendering properly when using OpenGL.
```

Comment:
The fix for this appears to be missing in this PR.

Reply:
It is already in develop, see commit message.

Reply:
That is because that fix is already merged: https://github.com/OpenRCT2/OpenRCT2/commit/710da039a4c2639620fcdc4fcd0cd20c413532fa

Reply:
Gotcha. Cheers!